ECLIPs/ECLIP-2.md

```diff
@@ -0,0 +1,50 @@
+# Add OpenRPC Service Discovery
+
+    ECLIP: 2
+    Title: Add OpenRPC Service Discovery
+    Author: Shane Jonas, Zachary Belford
+    Status: WIP
+    Type: Interface
+    Created: 2019-02-25
+
+# Abstract
+### What is this?
+
+This is a proposal to add [OpenRPC](https://github.com/open-rpc/spec) support by adding the method [`rpc.discover`](https://github.com/open-rpc/spec#service-discovery-method) to the projects [JSON-RPC](https://www.jsonrpc.org/specification) API to enable automation and tooling.
```

Comment:
So I think this needs to be fleshed out a little moar. What projects (or kinds of projects) is this specifically geared toward? Have you got a list of example projects?

Reply:
```suggestion
This is a proposal to add [OpenRPC](https://github.com/open-rpc/spec) support to existing and future Ethereum-familly RPC services by adding the method [`rpc.discover`](https://github.com/open-rpc/spec#service-discovery-method) to these projects's [JSON-RPC](https://www.jsonrpc.org/specification) APIs, enabling automation and tooling.
```

Reply:
`to these projects's`

should maybe read
`to this projects`, `this`, + dont need the `'s`

Reply:
... to add OpenRPC support to [...] services by adding the method `rpc.discover` to this projects APIs, enabling...
?